utils/checks.stanza

```diff
@@ -1116,13 +1136,13 @@ public pcb-check capacitor-component (c:JITXObject) :
       )
 
   #CHECK(
-    condition =            has-property?(c.capacitor),
+    condition =            has-property?(c.capacitance),
```

Comment:
this will cause anything with an capacitance property assigned to call this check vs a capacitor. desired behavior is for only capacitor components to be checked.

Reply:
Same reason as above -- the actual code does not run unless the `capacitor` property exists on the component in question.